python/pyarrow/_dataset_parquet.pyx

```diff
@@ -737,6 +737,8 @@ cdef class ParquetFragmentScanOptions(FragmentScanOptions):
             new CParquetFragmentScanOptions()))
         self.use_buffered_stream = use_buffered_stream
         self.buffer_size = buffer_size
+        if pre_buffer and not is_threading_enabled():
+            pre_buffer=False
```

Comment:
@jorisvandenbossche Is there a linter that can detect this style mismatch?

```suggestion
            pre_buffer = False
```

Reply:
> Is there a linter that can detect this style mismatch?

We use `cython-lint` which I would assume to catch this

Reply:
OK. We specify `--no-pycodestyle` explicitly. It may be related.

I've opened a new issue for it: GH-41908